server/container/metrics/metrics-es-reporter-v7/src/test/java/org/apache/james/metric/es/v7/ES6ReporterTest.java

```diff
@@ -20,10 +20,12 @@
 package org.apache.james.metric.es.v7;
 
 import org.apache.james.backends.es.v7.DockerElasticSearch;
+import org.apache.james.util.docker.Images;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
 class ES6ReporterTest extends ESReporterContract {
```

Comment:
Throwing a question here: do we really want to keep `ES2ReporterTest` and `ES6ReporterTest` here? What pushed us to keep `ES2ReporterTest` here in the first place? I forgot